squid/molecule/default/molecule.yml

```diff
@@ -20,8 +20,3 @@ provisioner:
           - dundee.ac.uk
 verifier:
   name: ansible
-  inventory:
```

Comment:
I have the defaults for squid set to probe example.com to keep the role generic. Not sure we actually want to probe example.com during a GitHub action though? Maybe it's fine instead of dundee.ac.uk

Reply:
Hmmm... doesn't it still use the original inventory?

Reply:
Ohh I see, that would be nice if true. 
Is the Action still running on individual branch commits or is it just at opening of a PR now?

Reply:
Thinking about it.... if it verifier didn't use the same inventory as `provisioner` the test should fail, since squid is configured to only allow dundee.ac.uk, not example.com. Unless somethings gone wrong....

Reply:
It should run on all pushes, and pull_requests. The latter is important if we get external PRs from someone who hasn't activated workflows on their fork.